features/networking/multus.feature

```diff
@@ -416,4 +416,232 @@ Feature: Multus-CNI related scenarios
     Then the output should contain "net1"
     And the output should contain "net2"
     And the output should contain 2 times:
-      | macvlan mode bridge |
+	    | macvlan mode bridge |
+
+ 
+  # @author anusaxen@redhat.com
+  # @case_id OCP-24490
+  @admin
+  Scenario: Pods can communicate each other with same vlan tag
+    # Make sure that the multus is enabled
+    Given the multus is enabled on the cluster
+    And I store all worker nodes to the :nodes clipboard
+    Given I have a project
+    When I run the :create admin command with:
+      | f | https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/networking/multus-cni/NetworkAttachmentDefinitions/bridge-host-local-vlan.yaml |
+      | n | <%= project.name %>                                                                                                                               |
+    Then the step should succeed 
+    #Labeling a worker node to make sure couple of future pods to be scheduled on this node only
+    When I run the :label admin command with:
+      | resource | node                    |
+      | name     | <%= cb.nodes[0].name %> |
+      | key_val  | test=worker1            |
+    Then the step should succeed
+    
+    Given I register clean-up steps:
+    """
+    When I run the :label admin command with:
+      | resource | node                    |
+      | name     | <%= cb.nodes[0].name %> |
+      | key_val  | test-                   |
+    Then the step should succeed
+    the step should succeed
+    """ 
+    #Labeing another worker node to make sure 3rd future pod to be scheduled on this node only
+    When I run the :label admin command with:
+      | resource | node                    |
+      | name     | <%= cb.nodes[1].name %> |
+      | key_val  | test=worker2            |
+    Then the step should succeed
+    
+    Given I register clean-up steps:
+    """
+    When I run the :label admin command with:
+      | resource | node                    |
+      | name     | <%= cb.nodes[1].name %> |
+      | key_val  | test-            	   |
+    Then the step should succeed
+    the step should succeed
+    """ 
+    #Creating first pod in vlan 100
+    When I run oc create over "https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/networking/multus-cni/Pods/generic_multus_pod_nodeselector.yaml" replacing paths:
+      | ["metadata"]["name"] | pod1-vlan100 |
+      | ["metadata"]["annotations"]["k8s.v1.cni.cncf.io/networks"]| bridgevlan100 |
+      | ["spec"]["nodeSelector"]["test"] | worker1 |
+    Then the step should succeed
+    And the pod named "pod1-vlan100" becomes ready
```

Comment:
This wont be specific time dependent. It will just poll until pod1-vlan100 become ready. It will time out with automation default timeout not sure how much is that.